### ISSUE TITLE
Page 5 - clicking page tabs on species list changes title to scientific name

### ISSUE THREAD

**Issue by jdrever** [jdrever/openwild#17]

**Comment by jdrever** [comment]
@tomdrever I don't quite follow this, are you ok to show me?

**Comment by tomdrever** [comment]
If you click from the species search page to the species list page using the common name it uses speciesNameToDisplay (e.g.
https://openwild.careful.digital/species/Hedera%20helix?page=1&speciesNameToDisplay=Ivy) but this isn't maintained by the page nav links.

**Comment by jdrever** [comment]
@tomdrever yes that's true - it is also a problem on ShropBot - see https://flora.shropshirebotany.org.uk/

I do think we need to fix it, but I'm not going to work on it as a priority because it is quite a decent sized change and hasn't yet been noticed as an issue in practice by ShropBot users
